### PR TITLE
feat: List known currencies

### PR DESCRIPTION
**Pull request by aliasbody**
Implemented the Currency API (there's only one method so it's a simple implementation) #17

### CODE REVIEW

**Comment by aliasbody** [comment]
@bennyn Does the following params need to be at an error state?
imagem

I fill like blocking the compilation just before of a keys order may be too extreme, and since it's an initial commit I don't know if this was made in purpose.

File: src/client/RESTClient.ts [hunk 1]
```diff
@@ -39,6 +40,7 @@ export class RESTClient extends EventEmitter {
   readonly product: ProductAPI;
   readonly profile: ProfileAPI;
   readonly user: UserAPI;
+  readonly currencies: CurrencyAPI;
```
Comment (bennyn) [inline src/client/RESTClient.ts:43]:
```suggestion
  readonly currency: CurrencyAPI;
```

Let's go with the "singular" form to be consistent.

**Comment by bennyn** [comment]
Hey @aliasbody, great to see you again. 👋 The "error" level for key sorting is on purpose but I am okay with switching it to "warn". We may need to adjust the prettier log level to see warnings coming from the Prettier ESLint plugin. I will do this in a separate PR.

File: src/client/RESTClient.ts (hunk 1 above)
Comment (aliasbody) [reply src/client/RESTClient.ts]: My bad. Fixed!

**Pull request merged**